### DISCUSSION TITLE
GitHub Codespaces cancelled when College enrolled in GitHub Campus

### DISCUSSION THREAD

**Discussion by cod-jacobs** [community/community#112289] [GitHub Education] [answered]
### Hi everyone,

I am unable to reverify my GitHub Teacher status after my College enrolled in the GitHub Campus program.

Students are using Codespaces to complete assignments, and that has been disrupted.

Here is the error message I am getting when attempting to reverify so that I can reenable Codespaces for my students:

I have reached out to my College's contact, but they say GitHub states that both of our email domains have been registered on GitHub's end.

What are my next steps?

Thank you.

**Comment by samus-aran** [comment, marked as answer]
Hi @cod-jacobs  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

Hope this helps! Let us know how you get along 👍
